SW_Flow.c

```diff
@@ -246,15 +246,15 @@ static void arrays2records(void) {
 	{
 		SW_Soilwat.swcBulk[Today][i] = lyrSWCBulk[i];
 		SW_Soilwat.drain[i] = lyrDrain[i];
-		SW_Soilwat.sTemp[i] = lyrsTemp[i];
+		SW_Soilwat.avgLyrTemp[i] = lyravgLyrTemp[i];
 		ForEachVegType(k)
 		{
 			SW_Soilwat.hydred[k][i] = lyrHydRed[k][i];
 			SW_Soilwat.transpiration[k][i] = lyrTransp[k][i];
 		}
 	}
-	SW_Soilwat.surfaceTemp = surfaceTemp[Today];
-	SW_Weather.surfaceTemp = surfaceTemp[Today];
+	SW_Soilwat.surfaceAvg = surfaceAvg[Today];
+	SW_Weather.surfaceAvg = surfaceAvg[Today];

```

Comment:
Could you please look into why we have average surface temperature both in `SW_Soilwat` and in `SW_Weather`? Would one of the two suffice?